src/raftstore/store/worker/split_check.rs

```diff
@@ -254,6 +271,30 @@ impl<C: Sender<Msg>> Runner<C> {
     }
 }
 
+fn cross_table(prev_key: &[u8], current_key: &[u8]) -> Option<Vec<u8>> {
```

Comment:
move the logic to the raftstore coprocessor.

Reply:
only coprocessor knows table

Reply:
Do you mean move this function to `mod coprocesser`, then call `coprocesser::cross_table` at split_checker?